Sources/protoc-gen-swift/FileGenerator.swift

```diff
@@ -370,5 +370,21 @@ class FileGenerator {
             p.outdent()
             p.print("]\n")
         }
+
+        generateVersionCheck(printer: &p)
```

Comment:
I took #35 to be without the structs – if you click on a type and jump to the definition, the first thing within it should be the real methods users will care about.  If we smash this block up at the top of with the generated comment, they sorta go together.

I'm not sure if swiftc does this, but at one point clang was giving up after too many errors, so my fear would be this error wouldn't get printed because of all the other things that this really is meant to cover.